targets/clojurescript/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var evalCLJS = require('./cljs');
+
+module.exports = function cljsToJs(resolve, reject, data) {
+  try {
+    evalCLJS(data.source, function(err, result) {
+      resolve({
+        errors: err || null,
+        result: result || null
+      });
+    });
+  } catch (e) {
+    console.log('failed ClojureScript');
+    resolve({
+      errors: [e],
```

Comment:
Is this error in the same format as all others, i.e. line number, character & message ([for example](https://github.com/jsbin/pennyworth/blob/master/targets/jade/index.js#L19-L22))